Doc/library/threading.rst

```diff
@@ -38,6 +38,30 @@ This module defines the following functions:
    returned.
 
 
+.. function:: excepthook(args, /)
+
+   Handle uncaught :func:`Thread.run` exception.
+
+   The *args* argument has the following attributes:
+
+   * *exc_type*: Exception type.
+   * *exc_value*: Exception value, can be ``None``.
+   * *exc_traceback*: Exception traceback, can be ``None``.
+   * *thread*: Thread which raised the exception, can be ``None``.
+
+   If *exc_type* is :exc:`SystemExit`, the exception is silently ignored.
+
+   If it raises an exception, :func:`sys.excepthook` is called to handle it.
```

Comment:
What is "it"? "If this function raises an exception", perhaps?

Reply:
done